tests/search-params.cc

```diff
@@ -22,18 +22,14 @@ using namespace nlohmann::literals;
 
 /* -------------------------------------------------------------------------- */
 
-  void
-printInput( const auto & pair )
-{
-  const std::string                               & name   = pair.first;
-  const flox::RegistryInput & params = pair.second;
-  std::cout << "    " << name << std::endl
-            << "      subtrees: "
-            << nlohmann::json( params.subtrees ).dump() << std::endl
-            << "      stabilities: "
-            << nlohmann::json(
-                 params.stabilities.value_or( std::vector<std::string> {} )
-               ).dump() << std::endl;
+/**
+ * @brief Ensure defaults/fallbacks work correctly with
+ *        @a flox::search::SearchParams `from_json`.
+ */
+  bool
+test_SearchParams_defaults0() {
```

Comment:
What is the point of this function if it always returns true? Also, is it ever used?

Reply:
Ah I forgot to remove it.

This was a test of the review system, and you passed 😬